GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -62,7 +62,7 @@ public sealed partial class RevisionGridControl : GitModuleControl, IScriptHostC
     {
         public event EventHandler<DoubleClickRevisionEventArgs>? DoubleClickRevision;
         public event EventHandler<FilterChangedEventArgs>? FilterChanged;
-        public event EventHandler<GitUIEventArgs>? RevisionGraphLoaded;
+        public event EventHandler<GridLoadingEventArgs>? RevisionGraphLoading;
```

Comment:
I suggest to retain the event to indicate the data is loaded as well. It may be useful for plugins.

```suggestion
        /// <summary>
        ///  Occurs whenever the revision graph has been populated with the data.
        /// </summary>
        public event EventHandler<DataLoadEventArgs>? DataLoaded;

        /// <summary>
        ///  Occurs whenever the revision graph has started loading the data.
        /// </summary>
        public event EventHandler<DataLoadEventArgs>? DataLoading;
```

Reply:
I fam a little hesitant to add features without a direct need, but can add this.